node/src/actors/json_rpc/json_rpc_methods.rs

```diff
@@ -65,7 +65,7 @@ pub fn jsonrpc_io_handler(
     //io.add_method("getOutput", |params: Params| get_output(params.parse()));
     io.add_method("syncStatus", |_params: Params| status());
     io.add_method("dataRequestReport", |params: Params| {
-        data_request_report(params.parse())
+        data_request_info(params.parse())
```

Comment:
No need to rename this method

Reply:
Agreed. I wanted to make it clear that it changed, but that may not have been the best approach.

Reply:
Actually, since the output of the command is exactly the same you could say that it hasn't changed, which is great for users.